tsconfig.json

```diff
@@ -1,5 +1,4 @@
 {
-  "extends": "../../tsconfig.json",
   "compilerOptions": {
     "module": "commonjs",
     "declaration": true,
```

Comment:
can we use es2020

Reply:
Sure, do you have a resource or a quick explanation on your preference?

Reply:
@vyasworks 
1. Do Node 16.* support es2020?
2. Do Lambda support Node 16?

Reply:
https://stackoverflow.com/a/67371788

Looks like es2020 won't be a problem for node 16

Reply:
A brief look [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) shows node  14 for lambdas which looks like it supports es2020 

https://stackoverflow.com/a/61305579

Reply:
Yes supported natively. That is a good SO thread too.

Reply:
we can use node14 - https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#tsconfig-bases

this is recommended config

Reply:
`"extends": "@tsconfig/node12/tsconfig.json",`
should we add that?

Reply:
No, sorry I meant - https://www.npmjs.com/package/@tsconfig/node14

There is a link to Node 14 in the above URL pls check that

Reply:
I should have waited to merge 🤕